docs/reference/aggregations/matrix/pca-aggregation.asciidoc

```diff
@@ -0,0 +1,158 @@
+[role="xpack"]
+[testenv="platinum"]
```

Comment:
Should it be `[testenv="basic"]` since the PR description says this feature is basic license?